content/en/platform/corda/4.12/community/getting-set-up.md

```diff
@@ -19,26 +19,24 @@ title: Getting set up for CorDapp development
 # Getting set up for CorDapp development
 
 There are four pieces of required software for CorDapp development:
-* Java 8 JDK
+* Java 17 JDK
 * IntelliJ IDEA
 * Git
 * Gradle
 
 {{< note >}} Visit the [platform support matrix]({{< relref "release-platform-support-matrix.md" >}}) to check the latest third-party application versions supported by Corda. {{< /note >}}
 
-## Installing the Java 8 JDK
+## Installing the Java 17 JDK
 
-Install the Java 8 JDK. Corda requires at least version 8u171, but do not currently support Java 9 or higher for this version of Corda.
+Install the Java 17 JDK. 
 
-Corda has been tested against the following Java builds:
+Corda 4.12 has been tested against the following Java builds:
```

Comment:
At what point does it become presumed that this page is talking about 4.12? im a bit wary about keeping consistency between versions, if we haven't mentioned the version before, should we start now?

(obviously up to you, just something I noticed)

Reply:
I thought it read better with the version but will remove for consistency.